lib/main_transactional.pm

```diff
@@ -0,0 +1,283 @@
+        loadtest 'installation/await_install';
+        loadtest 'installation/logs_from_installation_system';
+        loadtest 'installation/reboot_after_installation';
+        if (is_s390x) {
+            loadtest 'boot/reconnect_mgmt_console';
+            loadtest 'installation/first_boot';
+        } else {
+            loadtest 'microos/disk_boot';
+        }
+        loadtest 'console/textinfo';
+    }
+}
+
+sub load_autoyast_installation_tests {
+    loadtest 'autoyast/prepare_profile' if get_var('AUTOYAST_PREPARE_PROFILE');
+    loadtest 'installation/bootloader_start';
+    loadtest 'autoyast/installation';
+    loadtest 'autoyast/console';
+    loadtest 'autoyast/login';
+    loadtest 'autoyast/logs';
+    loadtest 'console/textinfo';
+}
+
+sub load_selfinstall_boot_tests {
+    loadtest 'installation/bootloader_uefi';
+    loadtest 'microos/selfinstall';
+}
+
+sub load_remote_target_tests {
+    loadtest 'installation/bootloader_start';
+    loadtest 'remote/remote_target';
+    loadtest 'console/textinfo';
+    loadtest 'microos/networking';
+    loadtest 'microos/libzypp_config';
+    loadtest 'microos/one_line_checks';
+    loadtest 'microos/services_enabled';
+    loadtest 'console/journal_check';
+    loadtest 'shutdown/shutdown';
+}
+
+sub load_remote_controller_tests {
+    loadtest 'support_server/login';
+    loadtest 'support_server/setup';
+    loadtest 'remote/remote_controller';
+    loadtest 'installation/welcome';
+    if (is_sle_micro) {
+        loadtest 'installation/scc_registration';
+    } elsif (is_microos) {
+        loadtest 'installation/online_repos';
+        loadtest 'installation/installation_mode';
+        loadtest 'installation/system_role';
+    }
+    loadtest 'installation/ntp_config_settings';
+    loadtest 'installation/user_settings_root';
+    loadtest 'installation/resolve_dependency_issues';
+    loadtest 'installation/installation_overview';
+    loadtest 'installation/disable_grub_timeout';
+    loadtest 'installation/start_install';
+    loadtest 'installation/await_install';
+    loadtest 'installation/reboot_after_installation';
+    loadtest 'support_server/wait_children';
+}
+
+sub load_common_tests {
+    loadtest 'console/regproxy' if is_regproxy_required;
+    loadtest 'microos/networking';
+    loadtest 'microos/libzypp_config';
+    loadtest 'microos/image_checks' if is_image;
+    loadtest 'microos/one_line_checks';
+    loadtest 'microos/services_enabled';
+    # MicroOS -old images use wicked, but cockpit-wicked is no longer supported in TW
+    loadtest 'microos/cockpit_service' unless is_staging || (is_microos('Tumbleweed') && get_var('HDD_1') =~ /-old/);
+    # Staging has no access to repos and the MicroOS-DVD does not contain ansible
+    # Ansible test needs Packagehub in SLE and it can't be enabled in SLEM
+    loadtest 'console/ansible' unless (is_staging || is_sle_micro);
+    loadtest 'console/kubeadm' if (check_var('SYSTEM_ROLE', 'kubeadm'));
```

Comment:
Do we run this test somewhere? I thought it is intended for Kubic or something that runs k8s.

Reply:
I wasn't sure if it's executed somewhere in O3, in some dev group or something, so I left it there just in case as it was in main.pm of `products/microos/main.pm`.

Reply:
Yeah, I have not found it myself. Let's see whether someone would have more insight. Otherwise, I would vote for removal as now it is a good opportunity to also clean up the schedule a bit.

Reply:
if you don't mind, I'll leave it here, otherwise I need to remove `tests/console/kubeadm.pm` to get green pipeline:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/actions/runs/3619223707/jobs/6100012477